plugins/amazonq/chat/jetbrains-community/resources/META-INF/plugin-chat.xml

```diff
@@ -34,7 +34,11 @@
             <reference id="aws.toolkit.open.q.window"/>
         </group>
         <!-- TODO: q.openchat will eventually be in amazonq, aws.toolkit.q.sign.in will eventually be in core. -->
-        <action id="q.openchat" class="software.aws.toolkits.jetbrains.services.amazonq.QOpenPanelAction"/>
+        <action id="q.openchat" class="software.aws.toolkits.jetbrains.services.amazonq.QOpenPanelAction">
+            <keyboard-shortcut keymap="Mac OS X" first-keystroke="control alt Q"/>
```

Comment:
now this adds both `control + alt + Q` and `cmd + alt + Q` in mac, needs more debugging